posts/2022-08-04-jakarta-core-profile-22009-beta.adoc

```diff
@@ -0,0 +1,270 @@
+
+Give it a try - and if you encounter any issues, please let us know!
+
+You can enable RESTful Web Services 3.1 by using the `restfulWS-3.1` feature in the `server.xml`. If you only need a RESTful client, then you can use the `restfulWSClient-3.1` feature instead.
+
+If you want to learn more about RESTful Web Services or see what is coming in future releases, please visit the link:https://projects.eclipse.org/projects/ee4j.rest[community page].
+
+
+[#json]
+== Jakarta JSON Binding and Processing
+The Jakarta JSON Processing specification allows users to parse and generate JSON strings. The Jakarta JSON Binding specification is built on top of JSON Processing and allows users to serialize and deserialize Java objects into JSON.
+
+The Jakarta JSON Processing specification has been updated with enhancements for handling duplicate keys when parsing JSON, and better handling of `java.lang.Number` superclass when generating JSON strings. The Jakarta JSON Binding specification has been updated to support polymorphism. 
+
+Open Liberty provides default implementations for both Jakarta JSON Processing and Binding. For JSON Processing we provide users with the Parsson implementation. For JSON Binding we provide users with the Yasson implementation.
+To get these default implementations you can enable the `jsonp-2.1` and/or `jsonb-3.0`` features.
+
+[source, xml]
+----
+<featureManager>
+    <feature>jsonb-3.0</feature>
+    <feature>jsonp-2.1</feature>
+</featureManager>
+----
+
+If you want to use a third-party Jakarta JSON Processing 2.1 or Binding 3.0 implementation you can do so by enabling the `jsonpContainer-2.1` and `jsonbContainer-3.0` features respectively. The JSON Container features require users to configure a Basic Extensions using Liberty Libraries (BELLs) element that points to their third-party implementation(s).
+
+[source, xml]
+----
+<featureManager>
+    <feature>jsonbContainer-3.0</feature>
+    <feature>jsonpContainer-2.1</feature>
+</featureManager>
+
+<library id="myJsonBindingProvider">
+  <file name="${path.to.json.binding.provider}"/>
+</library>
+
+<bell libraryRef="myJsonBindingProvider"/>
+
+<library id="myJsonProcessingProvider">
+  <file name="${path.to.json.processing.provider}"/>
+</library>
+
+<bell libraryRef="myJsonProcessingProvider"/>
+----
+
+NOTE: When you enable `jsonb-3.0` or `jsonbContainer-3.0` the `jsonp-2.1` feature is automatically enabled. However, enabling the `jsonpContainer-2.1` feature will supersede the `jsonp-2.1` feature and allow the user to provide a third-party implementation.
```

Comment:
Remove "NOTE:" - it's not necessary.
Add a comma after jsonbContainer-3.0.

I don't understand how to read the second sentence - it needs wording more clearly - I'm not sure what the point is that's being made?

Reply:
Maybe this information should just be provided in the relevant paragraphs above instead of being pulled out as a separate "note"?